src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  padding: 33px 17px 17px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  box-sizing: border-box;
+
+  transition: all 300ms;
```

Comment:
Consider specifying which properties you want to transition (e.g., `transform`, `color`) instead of using 'all', to avoid unnecessary performance overhead.